go/vt/mysqlctl/tmutils/schema.go

```diff
@@ -229,7 +231,11 @@ func SchemaDefinitionToSQLStrings(sd *tabletmanagerdatapb.SchemaDefinition) []st
 	sqlStrings := make([]string, 0, len(sd.TableDefinitions)+1)
 	createViewSQL := make([]string, 0, len(sd.TableDefinitions))
 
-	sqlStrings = append(sqlStrings, sd.DatabaseSchema)
+	// backtick database name since keyspace names appear in the routing rules, and they might need to be escaped
+	// unescape() first in case an already escaped string was passed
+	createDatabaseSQL := strings.Replace(sd.DatabaseSchema, "`{{.DatabaseName}}`", "{{.DatabaseName}}", -1)
```

Comment:
We should use `sqlescape.UnescapeID` here so that we have one spot to alter the unescaping if needed.

Reply:
Ah, right. Thanks!